src/config.cc

```diff
@@ -84,11 +84,15 @@ const char *configEnumGetName(configEnum *ce, int val) {
 
 Config::Config() {
   struct FieldWrapper {
-    const char *name;
+    const std::string name;
```

Comment:
```suggestion
    std::string name;
```

Reply:
Why do you make this change?

Reply:
`const std::string` may prevent `std::move` from FieldWrapper::name to key of `std::map` in https://github.com/apache/incubator-kvrocks/blob/8836bc57d18927534c928d0b8b7d19a5d93c9d5f/src/config.cc#L191